upf/upf.api

```diff
@@ -237,6 +237,7 @@ autoreply define upf_nwi_add_del {
   u32 ip6_table_id;
   u8 ipfix_policy[64];
   vl_api_address_t ipfix_collector_ip;
+  u32 ipfix_report_interval;
```

Comment:
So, this IPFIX-related fields won't be added into nwi_details?

Reply:
Thanks, forgot to add them there. Will post an update shortly

Reply:
Added